Decode/PerfItemValue.cs

```diff
@@ -966,7 +1166,8 @@ public string GetString()
                                 PerfConvert.Char32AppendWithControlChars(sb, this.GetU32(i), convertOptions);
                             }
                             break;
-                        case EventHeaderFieldFormat.IPv4:
+                        case EventHeaderFieldFormat.IPAddress:
+                        case EventHeaderFieldFormat.IPAddressObsolete:
```

Comment:
Note: Renamed IPv4/IPv6 to IPAddress/IPAddressObsolete. Either one works for either type of address, based on field size.